src/bb-di.php

```diff
@@ -156,10 +180,12 @@
     // $twig->addExtension(new Twig\Extension\OptimizerExtension());
     $twig->addExtension(new \Twig\Extension\StringLoaderExtension());
     $twig->addExtension(new Twig\Extension\DebugExtension());
-    $twig->addExtension(new Twig\Extensions\I18nExtension());
+    $twig->addExtension(new Symfony\Bridge\Twig\Extension\TranslationExtension());
     $twig->addExtension($box_extensions);
-    $twig->getExtension(Twig\Extension\CoreExtension::class)->setDateFormat($config['locale_date_format']);
-    $twig->getExtension(Twig\Extension\CoreExtension::class)->setTimezone($config['timezone']);
+    $twig->getExtension(Twig\Extension\CoreExtension::class)
```

Comment:
Not sure how other people feel, but I don't like moving this down to the next level like
IMO it hurts the readability of the code

Reply:
I think need github precommit hook in each commit! For to keep the code readable.